src/deepsparse/yolo/pipelines.py

```diff
@@ -236,10 +236,13 @@ def process_engine_outputs(
             ]  # post-processed values stored in first output
 
         # NMS
-        batch_output = postprocess_nms(
-            batch_output,
-            iou_thres=kwargs.get("iou_thres", 0.25),
-            conf_thres=kwargs.get("conf_thres", 0.45),
+        from ultralytics.yolo.utils import ops
+        import torch
+
+        batch_output = ops.non_max_suppression(
```

Comment:
From what I can tell the format of batch_output is the same as the old nms:

```python
            (x1, y1, x2, y2, confidence, class, mask1, mask2, ...).
```
So this should be good?

https://github.com/ultralytics/ultralytics/blob/f5dfd5be8b169935266ff602b8004a34d94feec5/ultralytics/yolo/utils/ops.py#L139

Reply:
Yep xyxy is YOLOv5 as well